backend/controllers/searchDownload.js

```diff
@@ -0,0 +1,35 @@
+const logger = require('../log')
+
+const downloadFactory = (searchApi, json2CsvParse) => {
+  const downloadResults = async (req, res) => {
+    const { user, groupCode, roleCode, status } = req.query
+
+    const searchResults = await searchApi({
+      locals: res.locals,
+      user,
+      groupCode,
+      roleCode,
+      status,
+      pageNumber: 0,
+      pageSize: 10000,
```

Comment:
This felt a bit hacky but it's a pageable endpoint so not sure of best way to do it - new endpoint?